src/Sulu/Bundle/ContentBundle/Resources/public/scss/modules/general.scss

```diff
@@ -2,14 +2,10 @@
     width: 95px;
 }
 
-#content-form {
+#content-form,
+.content-form {
```

Comment:
I think I am not fully understanding the implications of this: If you use this in another custom bundle, you use the class `content-form` now? So the only thing you do here is to save you from using an id? Shouldn't we find a more matching name, so that the class name makes more sense?

I guess by "avoiding BC breaks" you talk about javascript using the `content-form` id, right? I think the form relying on that could still be named like this, and we use for styling classes only. Then it wouldn't be necessary to have the `#content-form` selector, which might apply in a situation some developers do not think about it.